src/style.css

```diff
@@ -1 +1,46 @@
-/* add styles here */
+/* #region scale */
+
+.page__body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+/* #endregion */
+
+/* #region gray stars */
+.stars__star {
+  display: block;
+  width: 16px;
+  height: 16px;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-left: 4px;
+}
+
+/* #endregion */
+
+/* #region stars--1-5 */
+.stars--1 .stars__star:nth-child(-n + 1),
+.stars--2 .stars__star:nth-child(-n + 2),
+.stars--3 .stars__star:nth-child(-n + 3),
+.stars--4 .stars__star:nth-child(-n + 4),
+.stars--5 .stars__star:nth-child(-n + 5) {
+  background-image: url(images/star-active.svg); }
+
+/* #endregion */
+
+/* #region margins-zero */
+.stars--0 .stars__star:first-of-type,
+.stars--1 .stars__star:first-of-type,
+.stars--2 .stars__star:first-of-type,
+.stars--3 .stars__star:first-of-type,
+.stars--4 .stars__star:first-of-type,
+.stars--5 .stars__star:first-of-type {
+  margin-left: 0;
```

Comment:
it's actually not a good way to reset value like this. 
You can do much easier

```
.stars__star:not(:first-child) {
  margin-left: 4px;
}
```

and remove margin here
```
.stars__star {
  display: block;
  width: 16px;
  height: 16px;
  background-image: url(images/star.svg);
  background-repeat: no-repeat;
  background-position: center;
  margin-left: 4px;
}
```